v1/types/types.gunk

```diff
@@ -251,3 +251,156 @@ type Location struct {
 	// The longitude in degrees. It must be in the range [-180.0, +180.0]
 	Longitude float64 `pb:"2" json:"longitude"`
 }
+
+// Account type information
+type AccountInfoType int
+
+const (
+	UnknownAccountInfoType AccountInfoType = iota
+	TypeAccount
+	TypeProfile
+	TypeBank
+	TypeMsisdn
+)
+
+// Structure of account information
+type AccountInfo struct {
+
+	// Account Type
+	AccountInfoType AccountInfoType `pb:"1" json:"type"`
+
+	// Bank code
+	BankCode BankCode `pb:"2" json:"bank_code"`
+
+	// Account identifier
+	AccountID string `pb:"3" json:"account_id"`
+
+	// Type of identification code: account code, customer profile code or account number; depending on the type of account
+	Identifier string `pb:"4" json:"identifier"`
+}
+
+// Type of contact information
+type ContactDetailType int
+
+const (
+	// Unknown type of contact information
+	UnknownContactDetailType ContactDetailType = iota
+
+	// phone number
+	Phone
+
+	// Email address
+	Email
+
+	// Permanent address
+	ContactAddress
+
+	// Name
+	Name
+
+	// Short name / short name
+	Nickname
+
+	// Position
+	Position
+
+	// Day
+	Date
+
+	// Organization
+	Org
+
+	// Profile path
+	Url
+
+	// Image
+	Photo
+
+	// Note
+	Note
+
+	// Bank
+	Bank
+
+	// Card number
+	Card
+)
+
+// Label type marks contact information
+type ContactLabelType int
+
+const (
+	// Not available
+	UnknownContactLabelType ContactLabelType = iota
+
+	// Family
+	Home
+
+	// Task
+	Work
+
+	// Other
+	Custom
+)
+
+// Directory information structure
+type ContactDetail struct {
+	// Type of contact information
+	ContactDetailType ContactDetailType `pb:"1" json:"type"`
+
+	// Label type marks contact information
+	ContactLabelType ContactLabelType `pb:"2" json:"label"`
+
+	// Briefly programmed names for easy access
+	Name string `pb:"3" json:"name"`
+
+	// Value save details of contacts
+	Value string `pb:"4" json:"value"`
+
+	// Detailed data set related to contacts
+	Data map[string]string `pb:"5" json:"data"`
+}
+
+// Contact information structure
+type ContactInfo struct {
+
+	// The identifier of the user / customer in the phonebook
+	Identifier string `pb:"1" json:"identifier"`
+
+	// ID code of the contact information of the user / customer
+	ContactID string `pb:"2" json:"contact_id"`
+
+	// ID of customer / information
+	ProfileID string `pb:"3" json:"profile_id"`
+
+	// Briefly programmed names for easy access
+	Name string `pb:"4" json:"name"`
+
+	// The title / title displayed for a contact information.
+	Title string `pb:"5" json:"title"`
+
+	// Detailed data set related to contacts
+	Details []ContactDetail `pb:"6" json:"details"`
+
+	// Set of phone numbers associated with contact information
+	Phone map[string]bool `pb:"7" json:"phone"`
```

Comment:
why this use map[string]bool?
what represented with the bool?

Reply:
That's from the acb proto.

Reply:
Removed